remote-config.jsonc

```
@@ -84,6 +84,12 @@
 				},
                 {
                     "message": "You're invited to the DDEV Advisory Group, bimonthly on first Wednesday"
+                },
+                {
+                    "message": "Sign up for the DDEV newsletter! https://ddev.com/newsletter/"
+                },
+                {
+                    "message": "If you miss phpmyadmin in DDEV, `ddev get ddev/ddev-phpmyadmin`"
```

Capitalize proper noun, “run” commands:

```suggestion
                    "message": "If you miss phpMyAdmin in DDEV, run `ddev get ddev/ddev-phpmyadmin`"
```

---

Thank you! I moved to fast. Will get it on next round. Didn't know you were watching that closely, glad to have you.

---

I’ve been moving pretty slow and trying to catch up in little batches—just happened to catch this one quickly working through my notifications.

I mostly remove ddev/documentation review requests for things that’ve already been merged by the time I get to them.